Feature/helm chart (#186)
maykinmedia/objects-api: deployment/kubernetes/README.md

# Objects chart

> Here you can find a reference implementation of Objects deployment in the
> Kubernetes cluster via [Helm](https://helm.sh/).
> This Helm chart installs Objects API and is dependent on [PostgreSQL](https://github.com/bitnami/charts/tree/master/bitnami/postgresql)
> subchart.

Reviewer: ```suggestion

Here you can find a reference implementation of the Objects API deployment for
a Kubernetes cluster using [Helm](https://helm.sh/).

This Helm chart installs the Objects API and is dependent on a [PostgreSQL](https://github.com/bitnami/charts/tree/master/bitnami/postgresql)
database, installed using a subchart.
```

> :warning: The default settings are unsafe and should be used only for development purposes.

Reviewer: ```suggestion
:warning: The default settings are unsafe and should only be used for development purposes.
```

Configure proper secrets, enable persistence, add certificates before using in production.

## Installation

Install the Helm chart with following commands:

```bash
cd deployment/kubernetes/objects
helm dependency build .
helm install objects .
```

Use Kubernetes CLI to monitor the status of deployment:
```bash
kubectl get pods
```

> If ingress is not configured you can use `port-forward` to check the status of application:

Reviewer: ```suggestion
If the Ingress is not configured you can use `port-forward` to check the status of the application:
```
